cloudinit/config/schema.py

```diff
@@ -593,7 +618,7 @@ def load_doc(requested_modules: list) -> str:
     for mod_name in all_modules:
         if "all" in requested_modules or mod_name in requested_modules:
             (mod_locs, _) = importer.find_module(
-                mod_name, ["cloudinit.config"], ["schema"]
+                mod_name, ["cloudinit.config"], ["meta"]
```

Comment:
New-style schema definitions in cc_* modules will only have a "meta" attribute, not a local schema.